charms/istio-pilot/src/manifests/auth_filter.yaml.j2

```diff
@@ -39,6 +40,7 @@ spec:
                       {% for response_header in response_headers %}
```

Comment:
You already have a `{% for response_header in response_headers %}` loop in L34, why don't you re-use it?

Reply:
ohh this is an easy one to miss.  line 34 loops on `request_headers`, whereas this is `response_headers`

Reply:
Oh, even in the comment I had a hard time. Okay!